onnx_xla/translations/translate_concat.cc

```diff
@@ -0,0 +1,23 @@
+#include "onnx_xla/operator_registry.h"
+
+namespace onnx_xla {
+// Translate Concat Operator
+onnxStatus translateConcat(const Node& n,
+                           XlaBuilder& builder,
+                           ValueOpMap& valueToOp) {
+  if (!n.hasAttribute(kaxis)) {  // TODO: ENFORCE
+    std::cerr << "Missing required axis attribute" << std::endl;
+    return ONNXIFI_STATUS_INVALID_MODEL;
+  }
+  auto axis = n.i(kaxis);
+
+  std::vector<XlaOp> inputs;
+  for (auto i = 0; i < n.inputs().size(); ++i) {
+    inputs.emplace_back(valueToOp.at(n.inputs().at(i)));
```

Comment:
```
inputs.emplace_back(valueToOp.at(n.inputs()[i]));
```
as you don't need to do boundary check here?

Reply:
good point